pkgs/tools/inputmethods/ibus/wrapper.nix

```diff
@@ -7,7 +7,6 @@ let
   env = {
     buildInputs = [ ibus ] ++ plugins;
     nativeBuildInputs = [ lndir makeWrapper ];
-    propagatedUserEnvPackages = [ hicolor_icon_theme ];
```

Comment:
This should be changed to `propagatedUserEnvPkgs`. A loooong time ago, there was no such thing as `propagatedUserEnvPkgs` in the standard environment, so the KDE 4 packages implemented `propagatedUserEnvPackages` on its own. This probably continued to work until I removed the bulk of the KDE 4 packages.